src/Components/TodoContainer.js

```diff
@@ -0,0 +1,158 @@
+            fields: newTodo
+        }
+        console.log(todoData)
+        const options = {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`
+            },
+            body: JSON.stringify(todoData)
+        };
+
+
+        try {
+            const response = await fetch(url, options);
+
+            if (!response.ok) {
+                throw new Error(`Error: ${response.status} `);
+            };
+            await response.json();
+            fetchData()
+
+        } catch (error) {
+            console.error(error)
+
+        };
+    }
+
+
+    const deleteData = async (id) => {
+        const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/${id}`
+        const newTodos = todoList.filter((todo) => todo.id !== id);
+        setTodoList(newTodos);
+
+        const options = {
+            method: 'DELETE',
+            headers: {
+                Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`
+            },
+        };
+
+        try {
+            const response = await fetch(url, options);
+
+            if (!response.ok) {
+                throw new Error(`Error: ${response.status} `);
+            };
+            await response.json();
+            fetchData()
+
+        } catch (error) {
+            console.error(error)
+
+        };
+    };
+
+    useEffect(() => {
+        fetchData()
+
+    }, []);
+
+
+    useEffect(() => {
+        const stringifyTodoList = JSON.stringify(todoList);
+        if (!isLoading) {
+            localStorage.setItem('savedTodoList', stringifyTodoList)
+        }
+    }, [todoList, isLoading]);
+
+    const removeTodo = (id) => {
+        deleteData(id)
+    }
+    return (
+        (<div className={styles.titleInputListContainer}>
+            <BrowserRouter>
+                <Routes>
+                    <Route path="/" element={
+                        <>
+                            <div className={styles.titleInputContainer}>
+                                <h1 className={styles.title}>Shopping List</h1>
+                                <AddTodoForm onAddTodo={addTodo} />
+                            </div>
+                            {isLoading ? (
+                                <p>Loading...</p>
+                            ) : (
+                                <TodoList
+                                    todoList={todoList}
+                                    onRemoveTodo={removeTodo} />
+                            )}
+                        </>
+                    } />
+                    <Route path="/new" element={
+                        <>
+                            <h1>New Todo List</h1>
+                        </>
+                    } />
+                </Routes>
+            </BrowserRouter>
+        </div>)
+    );
```

Comment:
This JSX should be in App.js and the Route with path="/" need to have the element attribute equals to TodoContainer:
`<Route path="/" element={<TodoContainer />} />`

And this TodoContainer component will have the next JSX:

```
<>
    <div className={styles.titleInputContainer}>
        <h1 className={styles.title}>Shopping List</h1>
        <AddTodoForm onAddTodo={addTodo} />
    </div>
    {isLoading ? (
        <p>Loading...</p>
    ) : (
        <TodoList
            todoList={todoList}
            onRemoveTodo={removeTodo} />
    )}
</>
```

Reply:
Thank you Gonzalo! I was able to make the changes. Let me know if it's okay or if it still needs corrections. :)